src/GraphQL/Types/Composite/ComplexGraphType.cs

```diff
@@ -11,30 +11,50 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// Represents a complex (object) graph type.
```

Comment:
```suggestion
    /// Represents a base class for all complex (that is, having their own properties) input and output graph types.
```

Better not to mention `object` here so as not to confuse the reader.